src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,25 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
+
```

Comment:
Where is the constructor for this class? Also ensure you add header comments for all public classes.